docs/developing/testing.rst

```diff
@@ -0,0 +1,41 @@
+Testing
+#######
+
+This section covers writing tests for the ``h`` codebase.
+
+Getting started
+---------------
+
+Sean Hammond has written up a `guide to getting started`_ running and writing
+our tests, which covers some of the tools we use (``tox`` and ``pytest``) and
+some of the testing techniques they provide (factories and parametrization).
+
+.. _guide to getting started: https://www.seanh.cc/posts/running-the-h-tests
```

Comment:
I did consider that, but pretty much all of that guide would have been useful to me starting out, so there’s a good chance that most of it will be useful to any given new starter. Given that, I figured I’d link rather than copy it wholesale.